tests/func/StringVersionFactoryTest.php

```diff
@@ -46,34 +46,34 @@ public function getVersionSpecs(): array
                 null,
             ],
             [
-                rand(1, 9),
-                rand(1, 99),
-                rand(1, 999),
+                random_int(1, 9),
+                random_int(1, 99),
+                random_int(1, 999),
```

Comment:
The difference between `rand()` and `random_int()` is only in that the latter generates numbers that are _cryptographically secure_. This is unnecessary in this case, because we don't care about cryptographical security in this case. Besides not being relevant, `random_int()` is also slower.